### ISSUE TITLE
[Manure] Implement new Manure Treatments

### ISSUE THREAD

**Issue by ew3361zh** [RuminantFarmSystems/RuFaS#1620]
**Is your feature request related to a problem? Please describe.**
From the Manure Module Requirements, there are 2 requested new Manure Treatments that should be simulated/implemented: `Cap and Flair` and `Acidification`.

**Describe the solution you'd like**
Check the [Manure Module Requirements](https://docs.google.com/document/d/1qC-DjKE-EPVae1EuZAWVwCrYYLqOw5hscqszmzprSzE/edit) document for a starting point but Manure SMEs should be consulted for details on how these should be implemented.

**Additional context**
Issue #1619 is about distinguishing Manure Treatments from Manure Storages and rather than setting this up in the current Manure Treatment catch-all class, I suggest starting with #1619 to establish those distinct parent classes and having these inherit from the newly established Manure Treatment class.

**Comment by Isgotkowitz** [comment]
Grooming sessions: cap and flare is being implemented in #1857.

**Comment by Isgotkowitz** [comment]
Grooming session: @ew3361zh will follow up with SMEs to see if acidification should be its own treatment, or if it should be an added functionality to manure storages (similar to how cap and flare will work). Also the priority should be checked as well.

**Comment by ew3361zh** [comment]
From meeting with Elle:
- Would be great to have but needs design docs, heavy SME input, no urgent demand from anywhere, and no action currently for dev team to take.

Will resultingly move this to blocked.

**Comment by Isgotkowitz** [comment]
Grooming session: blocked by #2092

**Comment by allisterakun** [comment]
@elle-andreen any plans for this issue?

**Comment by elle-andreen** [comment]
@allisterakun Same status as Niko's 8/9/24 update, blocked by SME input. Although cap and flare has indeed been implemented.

**Comment by tomhuhh** [comment]
Groomed on Jan 30, 2026. A new issue (#2723) has been created specifically for implementing acidification.
